### DISCUSSION TITLE
How to filter templates with both wordpress and xss tags

### DISCUSSION THREAD

**Discussion by AaronChen0** [projectdiscovery/nuclei-templates#4546] [Q&A] [answered]
I want to scan only xss on wordpress sites. I try something like, `nuclei -tags wordpress -s medium -etags tag1,tag2,tag3 -l urls.txt`. 
Any better ways?

**Comment by ehsandeep** [comment, marked as answer]
@AaronChen0 unfortunately that's the only way as of now which is not convenient for sure, so this looks like a feature to support which allows AND based tags filtering where OR is being default which is the current behavior.

**Comment by ehsandeep** [comment]
GH issue for the support: https://github.com/projectdiscovery/nuclei/issues/2251

**Comment by ehsandeep** [comment]
@AaronChen0 this is now supported with latest nuclei.

```
nuclei -tc "contains(tags,'wordpress') && contains(tags,'xss')"
```
